tableauserverclient/models/user_item.py

```diff
@@ -238,6 +254,8 @@ def _set_values(
             self._auth_setting = auth_setting
         if domain_name:
             self._domain_name = domain_name
+        if idp_configuration_id:
```

Comment:
Do we have a doc comment mentioning that idp_configuration_id will override the domain name if both are present?

Reply:
@jacalata Good catch. That was a typo, and I removed it.